ui/cryptomaterial/tab_nav.go

```diff
@@ -0,0 +1,85 @@
+package cryptomaterial
+
+import (
+	"image"
+
+	"gioui.org/layout"
+	"gioui.org/op/clip"
+	"gioui.org/op/paint"
+
+	"github.com/crypto-power/cryptopower/ui/values"
+)
+
+type TabNavigation struct {
+	list          *ClickableList
+	selectedIndex int
```

Comment:
Add `selectedTabName string` here to. This will help reduce reliance on the index that could change if the tabs are swapped.